### PR TITLE
Day 3 Revisited

### PR DESCRIPTION
**Pull request by suprfrye**
I removed the view file and used string interpolation to display the db info. It shows up in the browser, but my tests fail. The error is that sea_lions does not exist.

Failure/Error: follow_redirect!
     ActiveRecord::StatementInvalid:
       PG::UndefinedTable: ERROR:  relation "sea_lions" does not exist
       LINE 5:                WHERE a.attrelid = '"sea_lions"'::regclass
                                                 ^
       :               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                            pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod
                       FROM pg_attribute a LEFT JOIN pg_attrdef d
                         ON a.attrelid = d.adrelid AND a.attnum = d.adnum
                      WHERE a.attrelid = '"sea_lions"'::regclass
                        AND a.attnum > 0 AND NOT a.attisdropped
                      ORDER BY a.attnum
@linkblaine @queerviolet

### CODE REVIEW

File: app/controllers/index.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'sinatra'
+
+get '/' do
+  "Welcome to the Sea Lions Den."
+end
+
+get '/sealions' do
+  sea_lions = ['Greg', 'Tiffany', 'Rosetta', 'Tommy']
```
Comment (linkblaine) [inline app/controllers/index.rb:8]: N/M this is not  a test.

File: spec/controllers/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  it 'should process post /sealions route' do
+    post '/sealions'
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response.body).to include("Let's welcome the new cubs:")
+  end
+
+  it 'should process get /sealions/:id route' do
+    get '/sealions/:id'
```
Comment (linkblaine) [inline spec/controllers/index_spec.rb:25]: Look at your get route here! You are not actually sending in a parameter. You have to format it like get '/sealions/#some known sealion id#

**Comment by linkblaine** [comment]
@suprfrye you probably need to run rake db:test:prepare to setup your test database. All of your code in the model looks good, although you're not persisting any data to the db! 

Add an Active Record backed Model and make the routes work with the saved data!

**Comment by suprfrye** [comment]
@linkblaine  thank you! I setup the test database and passed in actual objects and ... the tests passed!

**Comment by suprfrye** [comment]
Also weird thing is I could not use let(:sea_lion1) { SeaLion.create(name: "Nat", location: "NY, favorite_food: "rice cakes") }  to setup the tests.

**Pull request closed without merging**